### PR TITLE
Api 13371 v1 remove active status

### PR DESCRIPTION
**Pull request by JamesDanielBell-VA**
https://vajira.max.gov/browse/API-13371

- Removing activeStatus from V1

- Updating unit tests

### CODE REVIEW

File: facilities/src/main/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1.java [hunk 1]
```diff
@@ -10,6 +11,15 @@
 /** Utility class for transforming DatamartFacility to version 1 facility object and back. */
 @UtilityClass
 public final class FacilityTransformerV1 extends BaseVersionedTransformer {
+  /** Utility method for determining operatingStatus from Datamart activeStatus. */
+  private static OperatingStatus determineOperatingStatusFromActiveStatus(
```
Comment (giangnguyen2) [inline facilities/src/main/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1.java:15]: This function maybe conflicting or no longer need after [API-15738](https://github.com/department-of-veterans-affairs/lighthouse-facilities/pull/348)

File: facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV0Test.java [hunk 2]
```diff
@@ -626,6 +626,8 @@ public void losslessFacilityVisitorRoundtrip() {
                     FacilityTransformerV1.toFacility(
                         FacilityTransformerV0.toVersionAgnostic(facility), linkerUrl))))
         .usingRecursiveComparison()
+        .ignoringFields("attributes.detailedServices")
```
Comment (as-va) [inline facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV0Test.java:629]: Ignoring `detailedServices` twice?

Comment (as-va) [reply facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV0Test.java:629]: Additional fields don't need a new `ignoringFields` method - they can be added to the list in the existing one.

File: facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1Test.java [hunk 3]
```diff
@@ -189,7 +189,9 @@ public void datamartFacilityRoundtrip() {
         .isInstanceOf(AssertionError.class);
     assertThat(FacilityTransformerV1.toVersionAgnostic(facility))
         .usingRecursiveComparison()
-        .ignoringFields("attributes.detailedServices", "attributes.waitTimes")
+        .ignoringFields("attributes.detailedServices")
```
Comment (as-va) [inline facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1Test.java:192]: These can all be combined into a single method call.

File: facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1Test.java [hunk 4]
```diff
@@ -1050,6 +1051,8 @@ public void transformFacility() {
         .isInstanceOf(AssertionError.class);
     assertThat(FacilityTransformerV1.toVersionAgnostic(facility))
         .usingRecursiveComparison()
+        .ignoringFields("attributes.detailedServices")
```
Comment (as-va) [inline facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1Test.java:1054]: See above.

Comment (JamesDanielBell-VA) [reply facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1Test.java]: Updated!

File: facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1Test.java (hunk 3 above)
Comment (JamesDanielBell-VA) [reply facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV1Test.java]: Updated!

File: facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV0Test.java (hunk 2 above)
Comment (JamesDanielBell-VA) [reply facilities/src/test/java/gov/va/api/lighthouse/facilities/FacilityTransformerV0Test.java]: Updated!

**Review by as-va — APPROVED** [review]

**Comment by mmccollum-va** [comment]
@JamesDanielBell-VA please look to resolve the above merge conflicts

**Review by ChrisBratti-VA — APPROVED** [review]

**Review by mmccollum-va — APPROVED** [review]

**Review by as-va — CHANGES_REQUESTED** [review]
Please remove the `.DS_Store` file, then PR will be ready to merge.

**Review by as-va — APPROVED** [review]

**Review by as-va — CHANGES_REQUESTED** [review]
The merge of the base branch into this one added a new v1/CmsOverlayIT file which has references to ActiveStatus. That new file need to be updated to be compatible with the changes introduced in this PR; this branch is broken until then.

**Review by as-va — APPROVED** [review]

**Pull request merged**